### ISSUE TITLE
How to finetune change input InceptionResnetV1 to recieve 4 channels.

### ISSUE THREAD

**Issue by hvtrungagu** [timesler/facenet-pytorch#175]
``My project about recognize face. My dataset have 500 class with 70 image/class include one RGB Image(.png) and one Depthmap(.npy) of face. I colect depthmap by Camera Intel D435 as picture below. i use concate of numpy to combine after i trans to tensor 4D.
Now i want train model to extract features use Facenet with pytorch. How i can combine RGB image and depthmap to topology InceptionResnetV1 can train.
I tried with code: 
`resnet = InceptionResnetV1(
    classify=True,
    pretrained='vggface2',
    num_classes=len(dataset.class_to_idx)
).to(device)

resnet.conv2d_1a.conv=nn.Conv2d(4, 64, kernel_size=3, stride=2, padding=2, bias=False)` but can not
